.github/workflows/tests-canonical-extensions.yml

```
@@ -0,0 +1,34 @@
+name: Test canonical extensions
+
+on:
+  workflow_dispatch:
+    inputs:
+      woocommerce_version:
+        description: 'WooCommerce version to use for testing (as per https://qit.woo.com/wp-json/cd/v1/environment)?'
```

Very optional nitpick, but we can probably remove the `?` from each of these since they are inputs and not necessarily questions.

```suggestion
        description: 'WooCommerce version to use for testing (as per https://qit.woo.com/wp-json/cd/v1/environment)'
```

---

Nice catch! Addressed in d0fa95a71313317c3fa82505c5cb7200f81969a4.